README.md

```diff
@@ -1,2 +1,65 @@
-# Bookstore
-This is a project built with react that displays list of books where you can add and remove a selected book
+# React and Redux
+
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+
+[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
+
+[ESLint](https://eslint.org/) is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+    - **Remember** to use the file linked above
+    - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub Actions:
+
+![gh actions checks](../assets/images/gh-actions-eslint-stylelint-checks.png)
+
+Click on the `Details` link to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+### ESLint
+
+1. Run 
+    ```
+    npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+    ```
+    *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
```

Comment:
## [OPTION]
- [ ] Kindly make your README file more professional by following the microverse README template.